GSoC-2025/pranav_kapoor.md

```diff
@@ -0,0 +1,73 @@
+- **Full name:** Pranav Kapoor  
+- **Email:** pranav33317@gmail.com  
+- **GitHub username:** pranav33317  
+- **Zulip username:** Pranav Kapoor  
+- **Location & time-zone:** New Delhi, India (IST, GMT+5:30)  
+- **Personal website / project portfolio:** [github.com/pranav33317](https://github.com/pranav33317)  
+- **Code contribution:**  
+  - PR: [Disable parallel mesh creation (PR #546)](https://github.com/brainglobe/brainglobe-atlasapi/pull/546#pullrequestreview-2729541724)  
+    (I worked on disabling parallel mesh creation in various files while addressing [Issue #429](https://github.com/brainglobe/brainglobe-atlasapi/issues/429).)  
+- **Proposal discussion link:**  
+  - [Proposal Discussion (PR #554)](https://github.com/brainglobe/brainglobe-atlasapi/pull/554)  
```

Comment:
Please see your other PR. 

This should be a link to the discussion about your proposed project, i.e. this PR.